7-layouts/styles.css

```diff
@@ -0,0 +1,210 @@
+}
+
+.login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh;
+  padding: 22px;
+}
+
+.description {
+  height: 50vh;
+  background-image: url("./assets/bg-twitter.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.login__main {
+  padding: 14px;
+  max-width: 432px;
+}
+
+.login__icon img {
+  width: 4rem;
+}
+
+.login__title {
+  font-size: calc(1.5rem + 2vw);
+  font-weight: bold;
+}
+
+.login__button {
+  margin-block-start: 5rem;
+}
+
+.button {
+  display: block;
+  border-radius: 16px;
+  height: 3.3rem;
+  margin-block: 10px;
+  font-weight: 600;
+}
+
+.button--signup {
+  background-color: #39a2f3;
+  color: white;
+  width: 300px;
+  border: none;
+}
+
+.login__button .button--login {
+  margin-inline: auto;
+}
+
+.login__button .button--signup {
+  margin-inline: auto;
+}
+
+.button--login {
+  background-color: transparent;
+  border: 2px solid #39a2f3;
+  width: 300px;
+  color: #39a2f3;
+}
+
+.description__wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  padding: 25px;
+  background: rgb(2, 0, 36);
+  background: radial-gradient(
+    circle,
+    rgba(2, 0, 36, 0.5861695019804797) 0%,
+    rgba(2, 24, 71, 0.589) 52%
+  );
+}
+
+.description__list {
+  list-style: none;
+  font-size: calc(1.5rem + 0.7vw);
+  color: aliceblue;
+}
+
+.description__list--item {
+  display: flex;
+  align-items: center;
+  margin-block-end: 38px;
+}
+
+.description__list img {
+  margin-inline-end: 13px;
+  filter: invert(100%);
+  max-height: 32px;
+}
+
+.footer__buttons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 7rem;
+}
+
+.footer__buttons .button--signup {
+  width: 150px;
+  margin-inline: 15px;
+}
+
+.footer__buttons .button--login {
+  width: 150px;
+  margin-inline: 15px;
+}
+
+.footer__links {
+  width: 100vw;
+  height: 7rem;
+  background-color: white;
+  padding-block: 20px;
+  padding-inline: 10px;
+  text-align: center;
+}
+
+.footer__links .link {
+  text-decoration: none;
+  color: rgb(104, 103, 103);
+  margin-inline: 5px;
+}
+
+.login-optional {
+  display: none;
+}
+
+@media (min-width: 1000px) {
+  .login-optional {
+    position: relative;
+    top: -165px;
+    display: flex;
+    max-width: 600px;
+    margin: 0 auto;
+  }
+
+  .login-optional .button--login {
+    display: inline;
+    width: 65px;
+    margin-inline-start: 5px;
+  }
+
+  .login-optional__form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .login-optional__form-input {
+    border: none;
+    border-bottom: 2px solid rgb(124 124 124);
+    border-radius: 5px;
+    height: 45px;
+    background-color: rgb(231, 235, 238);
+    margin-inline: 8px;
+    padding: 5px;
+  }
+
+  .login-optional__form-input::placeholder {
+    position: relative;
+    bottom: 11px;
+  }
+}
+
+@media screen and (min-width: 700px) {
```

Comment:
This block should go in line 150, better to read from top to bottom what's being applied according to the media query.

Reply:
Taken note  👌